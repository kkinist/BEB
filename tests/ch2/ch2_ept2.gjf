%chk=ch2.chk
%nprocs=4
%mem=1000mw
# EPT(OVGF+P3,ReadOrbitals)/GEN gfinput geom=check guess=check scf=xqc pop(all,thresh=1)

BEB step 5: cation EPT for VIE2 for ch2

1 2

C H 0
6-311+G(d,p)
****

3 3 2 2

