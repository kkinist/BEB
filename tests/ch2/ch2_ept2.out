 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-26982.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     26984.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=ch2.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ----------------------------------------------------------------------
 # EPT(OVGF+P3,ReadOrbitals)/GEN gfinput geom=check guess=check scf=xqc
  pop(all,thresh=1)
 ----------------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1,116=-2/1,2,3;
 4/5=1/1;
 5/5=2,8=3,13=1,38=6/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/11=30100/8;
 6/7=2,8=2,9=2,10=2,87=-1,88=1/1;
 99/5=1,9=1/99;
 ---------------------------------------
 BEB step 5: cation EPT for VIE2 for ch2
 ---------------------------------------
 Structure from the checkpoint file:  "ch2.chk"
 Charge =  1 Multiplicity = 2
 Redundant internal coordinates found in file.
 C,0,0.,0.,0.0661340759
 H,0,0.,-0.9932409277,-0.3639822479
 H,0,0.,0.9932409277,-0.3639822479
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.066134
      2          1           0        0.000000   -0.993241   -0.363982
      3          1           0        0.000000    0.993241   -0.363982
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  C    0.000000
     2  H    1.082371   0.000000
     3  H    1.082371   1.986482   0.000000
 Stoichiometry    CH2(1+,2)
 Framework group  C2V[C2(C),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000000    0.000000    0.107529
      2          1           0        0.000000    0.993241   -0.322587
      3          1           0        0.000000   -0.993241   -0.322587
 ---------------------------------------------------------------------
 Rotational constants (GHZ):   1582.9318105    254.1515972    218.9909540
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   6 1.00       0.000000000000
      0.4563240000D+04  0.1966650249D-02
      0.6820240000D+03  0.1523060193D-01
      0.1549730000D+03  0.7612690966D-01
      0.4445530000D+02  0.2608010331D+00
      0.1302900000D+02  0.6164620782D+00
      0.1827730000D+01  0.2210060280D+00
 SP   3 1.00       0.000000000000
      0.2096420000D+02  0.1146600807D+00  0.4024869267D-01
      0.4803310000D+01  0.9199996477D+00  0.2375939567D+00
      0.1459330000D+01 -0.3030682134D-02  0.8158538515D+00
 SP   1 1.00       0.000000000000
      0.4834560000D+00  0.1000000000D+01  0.1000000000D+01
 SP   1 1.00       0.000000000000
      0.1455850000D+00  0.1000000000D+01  0.1000000000D+01
 SP   1 1.00       0.000000000000
      0.4380000000D-01  0.1000000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6260000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3386500000D+02  0.2549381454D-01
      0.5094790000D+01  0.1903731086D+00
      0.1158790000D+01  0.8521614860D+00
 S   1 1.00       0.000000000000
      0.3258400000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027410000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****

 There are    17 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     2 symmetry adapted cartesian basis functions of A2  symmetry.
 There are     6 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    10 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    16 symmetry adapted basis functions of A1  symmetry.
 There are     2 symmetry adapted basis functions of A2  symmetry.
 There are     6 symmetry adapted basis functions of B1  symmetry.
 There are    10 symmetry adapted basis functions of B2  symmetry.
    34 basis functions,    52 primitive gaussians,    35 cartesian basis functions
     4 alpha electrons        3 beta electrons
       nuclear repulsion energy         6.1332544348 Hartrees.
 NAtoms=    3 NActive=    3 NUniq=    2 SFac= 2.25D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    34 RedAO= T EigKep=  1.84D-02  NBF=    16     2     6    10
 NBsUse=    34 1.00D-06 EigRej= -1.00D+00 NBFU=    16     2     6    10
 Initial guess from the checkpoint file:  "ch2.chk"
 B after Tr=     0.000000    0.000000    0.000000
         Rot=    1.000000    0.000000    0.000000    0.000000 Ang=   0.00 deg.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1)
       Virtual   (B1) (A1) (B2) (A1) (B1) (A1) (B2) (A1) (B2) (B1)
                 (A1) (B2) (A1) (A2) (A1) (B1) (B2) (A1) (B1) (B2)
                 (A1) (A2) (B2) (B2) (A1) (A1) (B1) (A1) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2)
       Virtual   (A1) (B1) (A1) (B2) (A1) (B1) (A1) (B2) (A1) (B2)
                 (B1) (A1) (B2) (A1) (A2) (A1) (B1) (B2) (A1) (B1)
                 (B2) (A1) (A2) (B2) (B2) (A1) (A1) (B1) (A1) (B2)
                 (A1)
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 0.5000 <S**2>= 0.7660 S= 0.5080
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=1020339.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -38.5774287876     A.U. after   11 cycles
            NFock= 11  Conv=0.29D-08     -V/T= 1.9997
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 0.5000 <S**2>= 0.7543 S= 0.5021
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     0.7543,   after     0.7500
 QCSCF skips out because SCF is already converged.
 ExpMin= 4.38D-02 ExpMax= 4.56D+03 ExpMxC= 6.82D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    34
 NBasis=    34 NAE=     4 NBE=     3 NFC=     1 NFV=     0
 NROrb=     33 NOA=     3 NOB=     2 NVA=    30 NVB=    31
 Semi-Direct transformation.
 ModeAB=           4 MOrb=             3 LenV=    1048391354
 LASXX=        13664 LTotXX=       13664 LenRXX=       29211
 LTotAB=       15547 MaxLAS=       49599 LenRXY=           0
 NonZer=       42875 LenScr=      785920 LnRSAI=       49599
 LnScr1=      785920 LExtra=           0 Total=      1650650
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=1 Pass  1:  I=   1 to   3.
 (rs|ai) integrals will be sorted in core.
 ModeAB=           4 MOrb=             2 LenV=    1048391354
 LASXX=         9283 LTotXX=        9283 LenRXX=       17787
 LTotAB=        8504 MaxLAS=       33066 LenRXY=           0
 NonZer=       27070 LenScr=      785920 LnRSAI=       33066
 LnScr1=      785920 LExtra=           0 Total=      1622693
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=2 Pass  1:  I=   1 to   2.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.2739633179D-02 E2=     -0.8932469307D-02
     alpha-beta  T2 =       0.2193827570D-01 E2=     -0.6972530411D-01
     beta-beta   T2 =       0.6602742097D-03 E2=     -0.2061083655D-02
 ANorm=    0.1012589840D+01
 E2 =    -0.8071885707D-01 EUMP2 =    -0.38658147644685D+02
 (S**2,0)=  0.75425D+00           (S**2,1)=  0.75057D+00
 E(PUHF)=      -0.38579013643D+02        E(PMP2)=      -0.38659047823D+02
 GREENY: Electron Propagator Program
 FoFJK:  IHMeth= 0 ICntrl=     200 DoSepK=F KAlg= 0 I1Cent=   0 FoldK=F
 IRaf= 990000000 NMat=   1 IRICut=       1 DoRegI=T DoRafI=F ISym2E=-1.
 FoFCou: FMM=F IPFlag=           0 FMFlag=           0 FMFlg1=           0
         NFxFlg=           0 DoJE=F BraDBF=F KetDBF=F FulRan=T
         wScrn=  0.000000 ICntrl=     200 IOpCl=  1 I1Cent=           0 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.

 Summary of results for alpha spin-orbital   3  OVGF: 
 Koopmans theorem:            -0.85829D+00 au  -23.355 eV
 Converged second order pole: -0.82665D+00 au  -22.494 eV  0.955 (PS)
 Converged third  order pole: -0.82679D+00 au  -22.498 eV  0.944 (PS)
 Outer Valence Approximation: -0.82674D+00 au  -22.497 eV  0.939 (PS)

 Summary of results for alpha spin-orbital   3    P3: 
 Koopmans theorem:            -0.85829D+00 au  -23.355 eV
 Converged second order pole: -0.82665D+00 au  -22.494 eV  0.955 (PS)
 Converged 3rd order P3 pole: -0.82065D+00 au  -22.331 eV  0.944 (PS)

 Summary of results for beta  spin-orbital   2  OVGF: 
 Koopmans theorem:            -0.98760D+00 au  -26.874 eV
 Converged second order pole: -0.97562D+00 au  -26.548 eV  0.955 (PS)
 Converged third  order pole: -0.97969D+00 au  -26.659 eV  0.947 (PS)
 Outer Valence Approximation: -0.98106D+00 au  -26.696 eV  0.944 (PS)

 Summary of results for beta  spin-orbital   2    P3: 
 Koopmans theorem:            -0.98760D+00 au  -26.874 eV
 Converged second order pole: -0.97562D+00 au  -26.548 eV  0.955 (PS)
 Converged 3rd order P3 pole: -0.97224D+00 au  -26.456 eV  0.950 (PS)

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1)
       Virtual   (B1) (A1) (B2) (A1) (B1) (A1) (B2) (A1) (B2) (A1)
                 (B1) (B2) (A1) (A2) (A1) (B1) (B2) (A1) (B1) (B2)
                 (A1) (A2) (B2) (B2) (A1) (A1) (B1) (A1) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2)
       Virtual   (B1) (A1) (A1) (B2) (B1) (A1) (A1) (B2) (A1) (B2)
                 (B1) (A1) (B2) (A1) (A2) (A1) (B1) (B2) (A1) (B1)
                 (B2) (A1) (A2) (B2) (B2) (A1) (A1) (B1) (A1) (B2)
                 (A1)
 The electronic state is 2-A1.
 Alpha  occ. eigenvalues --  -11.74300  -1.30172  -1.01020  -0.85829
 Alpha virt. eigenvalues --   -0.31335  -0.12173  -0.07864  -0.06767  -0.05290
 Alpha virt. eigenvalues --   -0.01545   0.02920   0.26363   0.28211   0.38461
 Alpha virt. eigenvalues --    0.41914   0.49444   0.52904   0.89185   1.03396
 Alpha virt. eigenvalues --    1.06825   1.19062   1.37454   1.59825   1.67895
 Alpha virt. eigenvalues --    1.81945   1.91505   2.33866   2.43585   2.49725
 Alpha virt. eigenvalues --    2.59133   2.68422   2.83865   3.63236  24.45408
  Beta  occ. eigenvalues --  -11.71250  -1.21043  -0.98760
  Beta virt. eigenvalues --   -0.28077  -0.25244  -0.11634  -0.07666  -0.04777
  Beta virt. eigenvalues --   -0.04064  -0.00429   0.04951   0.27126   0.30047
  Beta virt. eigenvalues --    0.43451   0.48102   0.52093   0.55856   0.89864
  Beta virt. eigenvalues --    1.12620   1.14413   1.21549   1.38985   1.61294
  Beta virt. eigenvalues --    1.72526   1.84417   1.93002   2.34500   2.45001
  Beta virt. eigenvalues --    2.50324   2.63589   2.69987   2.90822   3.64465
  Beta virt. eigenvalues --   24.47665
          Condensed to atoms (all electrons):
               1          2          3
     1  C    4.862572   0.363604   0.363604
     2  H    0.363604   0.351864  -0.010357
     3  H    0.363604  -0.010357   0.351864
          Atomic-Atomic Spin Densities.
               1          2          3
     1  C    1.068986  -0.011316  -0.011316
     2  H   -0.011316  -0.014654   0.002793
     3  H   -0.011316   0.002793  -0.014654
 Mulliken charges and spin densities:
               1          2
     1  C    0.410221   1.046354
     2  H    0.294890  -0.023177
     3  H    0.294890  -0.023177
 Sum of Mulliken charges =   1.00000   1.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  C    1.000000   1.000000
 Electronic spatial extent (au):  <R**2>=             17.1778
 Charge=              1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=             -0.5373  Tot=              0.5373
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -4.2897   YY=             -2.6556   ZZ=             -5.3495
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -0.1915   YY=              1.4427   ZZ=             -1.2512
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=             -0.7496  XYY=              0.0000
  XXY=              0.0000  XXZ=             -0.0368  XZZ=              0.0000  YZZ=              0.0000
  YYZ=             -1.5419  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -2.9636 YYYY=             -6.0648 ZZZZ=             -6.0758 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -2.3126 XXZZ=             -1.4988 YYZZ=             -2.2702
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 6.133254434788D+00 E-N=-9.918487502445D+01  KE= 3.858724046699D+01
 Symmetry A1   KE= 3.635733899764D+01
 Symmetry A2   KE= 7.136171327049D-35
 Symmetry B1   KE= 9.547612803142D-33
 Symmetry B2   KE= 2.229901469350D+00

 Atomic contributions to Alpha molecular orbitals:
 Alpha occ 1 OE=-11.743 is C1-s=1.00
 Alpha occ 2 OE=-1.302 is C1-s=0.78 H3-s=0.10 H2-s=0.10
 Alpha occ 3 OE=-1.010 is C1-p=0.60 H2-s=0.19 H3-s=0.19
 Alpha occ 4 OE=-0.858 is C1-p=0.88 C1-s=0.05 H3-s=0.03 H2-s=0.03
 Alpha vir 5 OE=-0.313 is C1-p=0.99
 Alpha vir 6 OE=-0.122 is C1-s=1.01 C1-p=0.04 H2-s=-0.02 H3-s=-0.02
 Alpha vir 7 OE=-0.079 is C1-p=1.09 H2-s=-0.05 H3-s=-0.05
 Alpha vir 8 OE=-0.068 is C1-p=1.02
 Alpha vir 9 OE=-0.053 is C1-p=1.00
 Alpha vir 10 OE=-0.015 is H3-s=0.38 H2-s=0.38 C1-s=0.20 C1-d=0.02
 Alpha vir 11 OE=0.029 is H3-s=0.52 H2-s=0.52 C1-p=-0.06 C1-d=0.02
 Alpha vir 12 OE=0.264 is H3-s=0.49 H2-s=0.49 C1-p=0.14 C1-s=-0.14
 Alpha vir 13 OE=0.282 is C1-p=0.71 H3-s=0.13 H2-s=0.13 H3-p=0.01 H2-p=0.01
 Alpha vir 14 OE=0.385 is C1-p=0.89 H2-s=0.08 H3-s=0.08 C1-s=-0.03
 Alpha vir 15 OE=0.419 is C1-p=1.00
 Alpha vir 16 OE=0.494 is C1-p=0.64 H3-s=0.16 H2-s=0.16 C1-d=0.02
 Alpha vir 17 OE=0.529 is C1-s=1.07 H3-s=-0.08 H2-s=-0.08 H2-p=0.03 H3-p=0.03 C1-d=0.02
 Alpha vir 18 OE=0.892 is C1-d=0.50 H2-p=0.25 H3-p=0.25
 Alpha vir 19 OE=1.034 is C1-d=0.71 H3-p=0.14 H2-p=0.14
 Alpha vir 20 OE=1.068 is C1-d=0.77 H2-p=0.11 H3-p=0.11
 Alpha vir 21 OE=1.191 is H2-p=0.32 H3-p=0.32 C1-d=0.29 H2-s=0.04 H3-s=0.04
 Alpha vir 22 OE=1.375 is C1-d=0.72 H2-p=0.09 H3-p=0.09 H2-s=0.06 H3-s=0.06 C1-s=-0.01
 Alpha vir 23 OE=1.598 is H2-p=0.38 H3-p=0.38 C1-d=0.23 C1-p=0.01
 Alpha vir 24 OE=1.679 is C1-d=0.48 H3-p=0.19 H2-p=0.19 H3-s=0.09 H2-s=0.09 C1-p=-0.02
 Alpha vir 25 OE=1.819 is H2-p=0.26 H3-p=0.26 C1-d=0.26 H3-s=0.09 H2-s=0.09 C1-p=0.03 C1-s=0.02
 Alpha vir 26 OE=1.915 is C1-d=0.50 H2-p=0.25 H3-p=0.25
 Alpha vir 27 OE=2.339 is H2-s=0.37 H3-s=0.37 H2-p=0.16 H3-p=0.16 C1-d=-0.05 C1-p=-0.01
 Alpha vir 28 OE=2.436 is H3-p=0.30 H2-p=0.30 C1-p=0.24 C1-d=0.10 H2-s=0.03 H3-s=0.03
 Alpha vir 29 OE=2.497 is H2-p=0.36 H3-p=0.36 H2-s=0.19 H3-s=0.19 C1-d=-0.06 C1-p=-0.03
 Alpha vir 30 OE=2.591 is C1-p=0.50 C1-d=0.16 H2-s=0.10 H3-s=0.10 H2-p=0.06 H3-p=0.06 C1-s=0.03
 Alpha vir 31 OE=2.684 is C1-p=1.00
 Alpha vir 32 OE=2.839 is C1-p=0.53 C1-d=0.17 H3-s=0.08 H2-s=0.08 H2-p=0.05 H3-p=0.05 C1-s=0.04
 Alpha vir 33 OE=3.632 is C1-p=0.81 C1-d=0.13 H3-s=0.03 H2-s=0.03
 Alpha vir 34 OE=24.454 is C1-s=1.00

 Atomic contributions to Beta  molecular orbitals:
 Beta  occ 1 OE=-11.712 is C1-s=1.00
 Beta  occ 2 OE=-1.210 is C1-s=0.66 H2-s=0.14 H3-s=0.14 C1-p=0.04
 Beta  occ 3 OE=-0.988 is C1-p=0.56 H3-s=0.21 H2-s=0.21
 Beta  vir 4 OE=-0.281 is C1-p=0.98
 Beta  vir 5 OE=-0.252 is C1-p=0.81 C1-s=0.14 H3-s=0.02 H2-s=0.02
 Beta  vir 6 OE=-0.116 is C1-s=1.06 H2-s=-0.06 H3-s=-0.06 C1-p=0.05
 Beta  vir 7 OE=-0.077 is C1-p=1.13 H3-s=-0.07 H2-s=-0.07
 Beta  vir 8 OE=-0.048 is C1-p=1.00
 Beta  vir 9 OE=-0.041 is C1-p=0.94 C1-s=0.02 H2-s=0.02 H3-s=0.02
 Beta  vir 10 OE=-0.004 is H2-s=0.36 H3-s=0.36 C1-s=0.16 C1-p=0.10 C1-d=0.01
 Beta  vir 11 OE=0.050 is H3-s=0.55 H2-s=0.55 C1-p=-0.13
 Beta  vir 12 OE=0.271 is H3-s=0.54 H2-s=0.54 C1-s=-0.16 C1-p=0.07
 Beta  vir 13 OE=0.300 is C1-p=0.82 H2-s=0.08 H3-s=0.08 H2-p=0.01 H3-p=0.01
 Beta  vir 14 OE=0.435 is C1-p=1.00
 Beta  vir 15 OE=0.481 is C1-p=0.97 C1-s=-0.05 H2-s=0.03 H3-s=0.03
 Beta  vir 16 OE=0.521 is C1-p=0.60 H2-s=0.18 H3-s=0.18 C1-d=0.01 H2-p=0.01 H3-p=0.01
 Beta  vir 17 OE=0.559 is C1-s=1.10 H2-s=-0.08 H3-s=-0.08 H2-p=0.03 H3-p=0.03 C1-p=-0.01 C1-d=0.01
 Beta  vir 18 OE=0.899 is C1-d=0.49 H3-p=0.25 H2-p=0.25
 Beta  vir 19 OE=1.126 is C1-d=0.65 H2-p=0.16 H3-p=0.16 C1-p=0.01
 Beta  vir 20 OE=1.144 is C1-d=0.71 H2-p=0.14 H3-p=0.14
 Beta  vir 21 OE=1.215 is H3-p=0.35 H2-p=0.35 C1-d=0.26 H3-s=0.02 H2-s=0.02
 Beta  vir 22 OE=1.390 is C1-d=0.74 H2-p=0.08 H3-p=0.08 H2-s=0.06 H3-s=0.06 C1-s=-0.01
 Beta  vir 23 OE=1.613 is H2-p=0.35 H3-p=0.35 C1-d=0.29
 Beta  vir 24 OE=1.725 is C1-d=0.52 H2-p=0.16 H3-p=0.16 H3-s=0.10 H2-s=0.10 C1-p=-0.02
 Beta  vir 25 OE=1.844 is C1-d=0.29 H3-p=0.24 H2-p=0.24 H3-s=0.09 H2-s=0.09 C1-p=0.02 C1-s=0.02
 Beta  vir 26 OE=1.930 is C1-d=0.51 H2-p=0.25 H3-p=0.25
 Beta  vir 27 OE=2.345 is H3-s=0.36 H2-s=0.36 H2-p=0.19 H3-p=0.19 C1-d=-0.05 C1-p=-0.04
 Beta  vir 28 OE=2.450 is H2-p=0.27 H3-p=0.27 C1-p=0.27 C1-d=0.13 H3-s=0.03 H2-s=0.03
 Beta  vir 29 OE=2.503 is H2-p=0.35 H3-p=0.35 H3-s=0.20 H2-s=0.20 C1-d=-0.06 C1-p=-0.03
 Beta  vir 30 OE=2.636 is C1-p=0.37 C1-d=0.22 H3-s=0.12 H2-s=0.12 H2-p=0.07 H3-p=0.07 C1-s=0.04
 Beta  vir 31 OE=2.700 is C1-p=1.00
 Beta  vir 32 OE=2.908 is C1-p=0.66 C1-d=0.14 H3-s=0.06 H2-s=0.06 C1-s=0.03 H3-p=0.03 H2-p=0.03
 Beta  vir 33 OE=3.645 is C1-p=0.81 C1-d=0.14 H3-s=0.02 H2-s=0.02
 Beta  vir 34 OE=24.477 is C1-s=1.00
 
 1\1\GINC-GAMBA\SP\UOVGF-FC\Gen\C1H2(1+,2)\IRIKURA\17-May-2017\0\\# EPT
 (OVGF+P3,ReadOrbitals)/GEN gfinput geom=check guess=check scf=xqc pop(
 all,thresh=1)\\BEB step 5: cation EPT for VIE2 for ch2\\1,2\C,0,0.,0.,
 0.0661340759\H,0,0.,-0.9932409277,-0.3639822479\H,0,0.,0.9932409277,-0
 .3639822479\\Version=ES64L-G09RevD.01\State=2-A1\HF=-38.5774288\MP2=-3
 8.6581476\PUHF=-38.5790136\PMP2-0=-38.6590478\S2=0.754253\S2-1=0.75057
 4\S2A=0.750007\RMSD=2.876e-09\PG=C02V [C2(C1),SGV(H2)]\\@


 HEAVEN'S NET CASTS WIDE.
 THOUGH ITS MESHES ARE COARSE, NOTHING SLIPS THROUGH.

                                          -- LAO-TSU
 Job cpu time:       0 days  0 hours  0 minutes  6.9 seconds.
 File lengths (MBytes):  RWF=     19 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 12:52:36 2017.
